### PR TITLE
Landing page cleanup

### PR DESCRIPTION
**Pull request by sanscontext**
Add back feedback link, reorder landing page, remove old icons

cc @SvenDowideit @joaofnfernandes 

Signed-off-by: LRubin <lrubin@docker.com>

### CODE REVIEW

**Comment by joaofnfernandes** [comment]
LGTM.

**Comment by SvenDowideit** [comment]
LGTM

File: content/index.md [hunk 1]
```diff
@@ -148,6 +127,35 @@ keywords = ["Docker, documentation, manual, guide, reference, api"]
...
+<li>
+<div class="media_image">
+	<a href="mailto:feedback@docker.com?subject=Docker%20Feedback"><img src="/assets/images/chat.png" alt="chat icon"></a>
+</div>
+	<div class="media_content">
+	<div data-mh="mh_docker_projects">
+	<h3><a href="mailto:feedback@docker.com?subject=Docker%20Feedback">Feedback!</a></h3>
+		<p>
+    Questions? Suggestions? Spot a typo?! 😱<br/>
+    Email us at <a href="mailto:feedback@docker.com?subject=Docker%20Feedback">Feedback@docker.com</a>.
```
Comment (thaJeztah) [inline content/index.md]: caption should probably be lowercase `feedback` as well?

File: content/index.md [hunk 2]
```diff
@@ -68,20 +63,8 @@ keywords = ["Docker, documentation, manual, guide, reference, api"]
 	</div>
```
Comment (thaJeztah) [inline content/index.md]: oh, while you're at it, could you remove `-rc1` here?

Comment (sanscontext) [inline content/index.md]: @sfsmithcha should I kill the extra-long version number bit?

Comment (sfsmithcha) [inline content/index.md]: @sanscontext , please do. thanks.

File: content/index.md (hunk 1 above)
Comment (sanscontext) [inline content/index.md]: Lowercased the email address, I'm assuming that's what you'd meant.

File: content/index.md (hunk 2 above)
Comment (sanscontext) [inline content/index.md]: Done. :)

File: content/index.md (hunk 1 above)
Comment (thaJeztah) [inline content/index.md]: yes, thanks!

**Pull request merged**